spec/support/acceptance/configure_splunk_hec.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
Why is this necessary? Shouldn't testing the setup be included in the acceptance tests?

Reply:
Turns out this isn't neccesary. Thanks!